src/js/disability-benefits/686/config/form.js

```diff
@@ -1,42 +1,140 @@
+    fullName,
+    ssn
   },
   chapters: {
-    chapter1: {
-      title: 'Chapter 1',
+    veteranInformation: {
+      title: 'Veteran Information',
       pages: {
-        page1: {
-          path: 'first-page',
-          title: 'First Page',
+        veteranInformation: {
+          path: 'veteran-information',
+          title: 'Veteran Information',
           uiSchema: {
+            fullName: fullNameUI,
+            ssnOrVa: _.merge({}, ssnOrVaUI, {
+              'ui:title': 'Social Security number or VA file number',
+              'ui:errorMessages': {
+                pattern: 'Your VA file number must be between 7 to 9 digits'
+              }
+            }),
+            relationship: {
+              'ui:title': 'Relationship to Veteran',
+              'ui:widget': 'radio',
+              'ui:options': {
+                labels: relationshipLabels
+              }
+            },
+            'view:applicantInfo': {
+              fullName: {
+                'ui:title': 'Applicant Information',
+                first: {
+                  'ui:title': 'First name',
+                  'ui:required': (field) => field.relationship !== 'veteran'
+                },
+                last: {
+                  'ui:title': 'Last name',
+                  'ui:required': (field) => field.relationship !== 'veteran'
+                },
+                middle: {
+                  'ui:title': 'Middle name'
+                },
+                suffix: {
+                  'ui:title': 'Suffix',
+                  'ui:options': {
+                    widgetClassNames: 'form-select-medium'
+                  }
+                }
+              },
+              ssn: _.assign(ssnUI, {
+                'ui:title': 'Social Security number',
+                'ui:required': (field) => field.relationship !== 'veteran'
+              }),
+              address: address.uiSchema('', false, (formData) => formData.relationship !== 'veteran'),
+              claimantEmail: {
+                'ui:title': 'Email address',
+                'ui:required': (field) => field.relationship !== 'veteran'
+              },
+              'ui:options': {
+                expandUnder: 'relationship',
+                expandUnderCondition: (field) => field === 'spouse' || field === 'child' || field === 'other'
+              }
+            },
           },
           schema: {
             type: 'object',
+            required: ['ssnOrVa', 'relationship'],
             properties: {
+              fullName,
+              ssnOrVa: {
+                type: 'string',
+              },
+              relationship: {
+                type: 'string',
+                'enum': [
+                  'veteran',
+                  'spouse',
+                  'child',
+                  'other'
+                ]
+              },
+              'view:applicantInfo': {
+                type: 'object',
+                properties: {
+                  fullName: {
```

Comment:
I think you want this to be `claimantFullName`, since that's the other name property in the schema: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/dist/21-686C-schema.json#L381

You can import it from `fullSchema.properties` and you won't need to define all the first/middle/last fields in here.